_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -3451,6 +3466,154 @@ public static function activate_akismet() {
 		}
 	}
 
+	/**
+	 * Install a specific plugin.
+	 *
+	 * @since 8.9.0
+	 *
+	 * @param WP_REST_Request $request {
+	 *     Array of parameters received by request.
+	 *
+	 *     @type string $slug Plugin slug.
+	 * }
+	 *
+	 * @return WP_REST_Response A response indicating whether or not the installation was successful.
+	 */
+	public static function install_plugin( $request ) {
+		$plugin = stripslashes( $request['slug'] );
+
+		jetpack_require_lib( 'plugins' );
+		$result = Jetpack_Plugins::install_plugin( $plugin );
+
+		if ( is_wp_error( $result ) ) {
+			return rest_ensure_response(
+				array(
+					'code'    => 'failure',
+					'message' => esc_html(
```

Comment:
We probably want to return a `WP_Error` object here.

Reply:
👍 Done in 8a27bc6775f94657c93558ff60f4d9d8f1d57e65